### PR TITLE
Spike/storybook

### PR DESCRIPTION
**Pull request by f1lander**
### Summary
Enables Storybook into our project, the `Button` story is just an example, new stories will be added in another PR's.

BTW, I guess I can add the vercel config to build the storybook too.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -111,6 +113,16 @@
     "extends": [
       "react-app",
       "react-app/jest"
+    ],
```
Comment (levity) [inline package.json:116]: does this work if you add it to `.eslintrc` instead? we should keep all rules there if possible

**Review by levity — COMMENTED** [review]
did you mean to check in some story files? i don't see anything relating to Button in the diff

**Comment by f1lander** [comment]
Sorry I forgot to push that change, I'll push later

File: package.json (hunk 1 above)
Comment (f1lander) [reply package.json:116]: Let me try, I guess this was added automatically

**Review by exrhizo — DISMISSED** [review]
LGTM, let's merge!

File: tsconfig.json [hunk 2]
```diff
@@ -15,7 +15,11 @@
     "isolatedModules": true,
     "noEmit": true,
     "jsx": "react-jsx",
-    "baseUrl": "src"
+    "baseUrl": "src",
+    "paths": {
+      "@emotion/core": ["./types/emotion-core-stub.d.ts"],
```
Comment (exrhizo) [inline tsconfig.json:20]: would it be useful to install emotion types?

**Review by exrhizo — APPROVED** [review]

**Pull request merged**